keel-sql/src/main/resources/db/changelog/20200515-give-event-table-a-pk.yml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+- changeSet:
+    id: give-event-table-a-pk
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: event
+        columns:
+        - name: uid
+          type: varchar(255)
+          constraints:
+            nullable: true
+    - modifySql:
+        dbms: mysql
+        append:
+          value: " first"
```

Comment:
Add a new `uid` column. Currently it will be nullable. We'll do a 2nd migration once every row has a value assigned.